VideoApp/VideoApp/Stores/AppSettings/APIEnvironment.swift

```diff
@@ -16,7 +16,7 @@
 
 import Foundation
 
-enum APIEnvironment: String, CaseIterable, Codable {
+enum Environment: String, CaseIterable, Codable {
```

Comment:
We should rename this file to `Environment.swift`.

Reply:
Right thanks!